### PR TITLE
Adds/updates docs for the FromServicesAttribute

### PR DESCRIPTION
**Pull request by Shazwazza**
Adds/updates some docs for the FromServicesAttribute with examples

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.ModelBinding/BinderMetadata/FromServicesAttribute.cs [hunk 1]
```diff
@@ -8,8 +8,60 @@ namespace Microsoft.AspNet.Mvc
 {
     /// <summary>
     /// This attribute is used on action parameters or model properties to indicate that
-    /// they will be bound using service provider.
+    /// they will be bound with the <see cref="ServicesModelBinder"/> using the <see cref="IServiceProvider"/>.
     /// </summary>
+    /// <example>
```
Comment (yishaigalatzer) [inline src/Microsoft.AspNet.Mvc.ModelBinding/BinderMetadata/FromServicesAttribute.cs]: The example has too many unrelated items, which makes it very long and not very useful in an intellisense window.
It is also hard to keep this sample in sync with the real recommended user code if for example registration APIs change over time.

We are going to have a short discussion in the team, and get back to you with recommendations.

Comment (Shazwazza) [inline src/Microsoft.AspNet.Mvc.ModelBinding/BinderMetadata/FromServicesAttribute.cs]: I hadn't considered intellisense, I added a couple examples just for reference and/or for MSDN style generated docs. If examples show up in intellisense then there's certainly not much point in putting detailed examples in code comments. Happy to remove these examples or update based on your recommendations and just wait until the markdown docs repo is available.

**Comment by yishaigalatzer** [comment]
@danroth27 what do we want to do on docs for this?

**Comment by yishaigalatzer** [comment]
@Shazwazza we haven't forgotten about this, still figuring out the doc story.

**Comment by davidfowl** [comment]
What's the deal with this @rynowak @danroth27 @Eilon

**Comment by Eilon** [comment]
@rynowak please take a look and review/merge/close, thanks!

**Comment by rynowak** [comment]
87c15bb6dc8fbb7672a6a5533a78de86cc23186f

**Pull request closed without merging**